packages/ui/ViewManager/tests/ViewManager-specs.js

```diff
@@ -1,12 +1,20 @@
 import '@testing-library/jest-dom';
-import {render, screen, waitFor} from '@testing-library/react';
+import {act, render, screen, waitFor} from '@testing-library/react';
 import {Component} from 'react';
 
 import ViewManager from '../';
 import {MockArranger} from './test-utils';
 
 describe('ViewManager', () => {
 	// Suite-wide setup
+	beforeEach(() => {
+		jest.useFakeTimers();
+	});
+
+	afterEach(() => {
+		jest.useRealTimers();
+	});
+
```

Comment:
How about move this "useFakeTimers" to TC.

I think it is required on one TC

Reply:
Since setTimeout is used in the Mock Arranger function, it is required for all test cases that use the Mock Arranger.